File: DigitalLibrary/Models/Book.cs
using System;

namespace DigitalLibrary.Models
{
    public class Book
    {
        public int Id { get; set; }
        public string CoverImage { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string Keywords { get; set; }
    }
}

File: DigitalLibrary/Scripts/Models/book.js
var app = app || {};

app.Book = Backbone.Model.extend({
    defaults: {
        coverImage: "../../Images/placeholder.png",
        title: "No title",
        author: "Unknown",
        releaseDate: "Unknown",
        keywords: "None"
    },
    //url:'/books',
    
    parse: function(response) {
        response.coverImage = response.CoverImage;
        response.title = response.Title;
        response.author = response.Author;
        response.releaseDate = response.ReleaseDate;
        response.keywords = response.Keywords;
        response.id = response.Id;
        return response;
    }
});
